### PR TITLE
Fix EZP-24393: Implement FieldDefinition form mapper for ezdate

### PR DESCRIPTION
**Pull request by glye**
Manual testing. This is based on eztime, but is even simpler.

https://jira.ez.no/browse/EZP-24393

### CODE REVIEW

File: lib/FieldType/Mapper/DateFormMapper.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class DateFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'defaultType',
+                'choice',
+                [
+                    'choices' => array(
```
Comment (pspanja) [inline lib/FieldType/Mapper/DateFormMapper.php]: We should be consistent with using short array syntax.

**Comment by pspanja** [comment]
+1

**Comment by glye** [comment]
Short array syntax done.

**Comment by glye** [comment]
Rebased after ezcountry commit

**Comment by lolautruche** [comment]
+1

**Comment by bdunogier** [comment]
+1

**Comment by glye** [comment]
Rebased on master.

**Pull request merged**